ansible_collections/arista/avd/plugins/modules/topology_generator.py

```diff
@@ -0,0 +1,39 @@
+# GNU General Public License v3.0+
```

Comment:
Please remove this GPL license (cf PR https://github.com/aristanetworks/ansible-avd/pull/2010)